### ISSUE TITLE
xcms modifying 'library' during R CMD check

### ISSUE THREAD

**Issue by jotsetung** [sneumann/xcms#240]
That is related to the email from Valerie Obenchain in the Bioc-devel mailing list. `xcms` is writing the results of the `Runit` unit tests to a summary file. Apparently that can be problematic, so we should fix that (i.e. don't write the runit summary).

**Comment by sneumann** [comment]
Indeed, as seen in https://github.com/kalibera/rlibwrite/blob/master/xcms.out#L179
```
* checking tests ... ERROR
  Running ‘doRUnit.R’
Running the tests in ‘tests/doRUnit.R’ failed.
Last 13 lines of output:
    cannot open the connection
  Calls: printTextProtocol -> cat -> file
...
  6: In file(file, ifelse(append, "a", "w")) :
    cannot open file '/var/scratchro/tomas/QA/R-73707/lib/xcms/unitTests/reportSummary.txt': Read-only file system
  Execution halted
```
which is called in https://github.com/sneumann/xcms/blob/devel/tests/doRUnit.R#L104
I already asked @kalibera for advice, as Rdisop has the same code. Happy to implement 
whatever the recommendation here. Yours, Steffen
